scripts/generator/graphql_generator/csharp/SDK/CartLineItems.cs.erb

```diff
@@ -376,10 +362,39 @@ namespace <%= namespace %>.SDK {
             return Delete(variantId);
         }
 
-        private void OnCartLineItemChange(CartLineItem modifiedLineItem) {
-            if (!LineItemsNew.Contains(modifiedLineItem) && !LineItemsModified.Contains(modifiedLineItem)) {
-                LineItemsModified.Add(modifiedLineItem);
+        public void UpdateLineItemsFromCheckoutLineItems(List<CheckoutLineItem> checkoutLineItems) {
+            foreach(CheckoutLineItem checkoutLineItem in checkoutLineItems) {
+                bool didUpdateID = false;
+                string checkoutVariantId = checkoutLineItem.variant().id();
+                string checkoutLineItemId = checkoutLineItem.id();
+                
+                foreach(CartLineItem cartLineItem in LineItems) {    
+                    if (cartLineItem.VariantId == checkoutVariantId) {
+                        cartLineItem.ID = checkoutLineItemId;
+                        didUpdateID = true;
+                        break;
+                    }
+                }
+
+                if (!didUpdateID) {
+                    long quantity = checkoutLineItem.quantity();
+                    Dictionary<string, string> customAttributes = null;
+
+                    try {
+                        List<<%= namespace %>.Attribute> attributes = checkoutLineItem.customAttributes();
```

Comment:
Seems weird that we have to wiggle back up a namespace here, why are these attributes not nested under the SDK namespace so we dont have to specify this whole path? Alternatively, we can just import the namespace at the top of the file...